plugins/services/src/js/utils/ServiceTableUtil.js

```diff
@@ -1,8 +1,11 @@
 import compareVersions from "compare-versions";
+import sort from "array-sort";
 
 import StatusSorting from "../constants/StatusSorting";
 import Framework from "../structs/Framework";
+/* eslint-disable no-unused-vars */
```

Comment:
Because we use it, for example - https://github.com/dcos/dcos-ui/blob/9a92c2c482fda6aa5de0c86cf4f7a74a37894859/plugins/services/src/js/utils/ServiceTableUtil.js#L81 .